includes/admin/class-wcmypa-admin.php

```diff
@@ -917,6 +931,116 @@ private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     }
 
     /**
+     * Output the chosen delivery options or the chosen pickup options.
+     *
+     * @param DeliveryOptions $deliveryOptions
+     *
+     * @return array[]|null
+     * @throws \Exception
+     */
+    private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
+    {
+        $signatureTitle     = (new WCMP_Checkout)->getDeliveryOptionsTitle(WCMYPA_Settings::SETTING_SIGNATURE_TITLE);
```

Comment:
Volgens mij doen we altijd parens `()` bij een constructor, en nu ben je dit ding 2x aan het aanmaken. Kun je `new WCMP_Checkout()` in een variabele stoppen? En misschien te out of scope, maar moet `getDeliveryOptionsTitle` niet gewoon static zijn?